docs/advanced/ast.mdx

```diff
@@ -1,11 +1,16 @@
-import { Button } from '@rebass/emotion'
+import {Button} from '@rebass/emotion'
```

Comment:
Actually I prefer the previous one personally which seems more readable.
Shall/Should we change `bracketSpacing` config of `prettier`?

@ChristianMurphy @wooorm @johno

Reply:
And about `trailingComma`, `all` is much better for diffing between commits, or `es5` for compatibility.

Reply:
I personally prefer the “current” style for both (as in: no spaces in braces and no trailing commas)

Reply:
That's OK, we can discuss it in the future of not, let's just keep it as is in this PR.😀